meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx/0001-FIO-internal-arch-arm64-dts-imx8mp-enable-I2C5-bus.patch

```diff
@@ -0,0 +1,63 @@
+From b544a83456cd47bebadc20a908b0b78668a613ed Mon Sep 17 00:00:00 2001
+From: Vanessa Maegima <vanessa.maegima@foundries.io>
+Date: Tue, 29 Mar 2022 15:44:59 -0300
+Subject: [PATCH] [FIO internal] arch: arm64: dts: imx8mp: enable I2C5 bus
+
+Enable I2C5 bus to connect SE050. I2C5 shares pins with flexcan1, so
+disable the latter.
+
+Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
+---
+ arch/arm64/boot/dts/freescale/imx8mp-evk.dts | 22 +++++++++++++++++++-
+ 1 file changed, 21 insertions(+), 1 deletion(-)
+
+diff --git a/arch/arm64/boot/dts/freescale/imx8mp-evk.dts b/arch/arm64/boot/dts/freescale/imx8mp-evk.dts
+index 0ece8e524e44..d326e5eb4c4f 100644
+--- a/arch/arm64/boot/dts/freescale/imx8mp-evk.dts
++++ b/arch/arm64/boot/dts/freescale/imx8mp-evk.dts
+@@ -302,7 +302,7 @@ &flexcan1 {
+ 	pinctrl-names = "default";
+ 	pinctrl-0 = <&pinctrl_flexcan1>;
+ 	xceiver-supply = <&reg_can1_stby>;
+-	status = "okay";
++	status = "disabled";
+ };
+ 
+ &flexcan2 {
+@@ -567,6 +567,19 @@ ov5640_mipi_1_ep: endpoint {
+ 	};
+ };
+ 
++&i2c5 {
++	clock-frequency = <100000>;
++	pinctrl-names = "default";
++	pinctrl-0 = <&pinctrl_i2c5>;
++	pinctrl-assert-gpios = <&pca6416 2 GPIO_ACTIVE_HIGH>; /* set high to select i2c5 instead of flexcan1 */
++	status = "okay";
++
++	se050@48 {
++		compatible = "nxp,se050";
++		reg = <0x48>;
++	};
```

Comment:
Is this node (se050) used by anyone? Because the element is enabled/used in u-boot / op-tee, so unclear if we need this enabled at the kernel side.

We should probably also apply this patch only when MACHINE_FEATURES has se05x defined.

Reply:
right, the node is not needed